.github/workflows/_bazel-build-test.yml

```diff
@@ -164,13 +164,19 @@ jobs:
           )
           docker exec -t "${container_name}" sh -c 'sudo chown -R jenkins . && sudo chown -R jenkins /dev && .jenkins/pytorch/test.sh && cp -Lr ./bazel-testlogs ./test/test-reports'
 
+      - name: Print undeleted logs
+        shell: bash
+        if: always()
+        run: |
+          cat test/**/*.log || exit 0
```

Comment:
Nit: This works but looks a bit weird to me. I normally see `cat test/**/*.log || true`